workspaces/azureml_devtestlabs/porter.yaml

```diff
@@ -3,18 +3,17 @@ version: 0.1.0
 description: "A Azure TRE workspace with Azure Machine Learning and Dev Test Labs"
 registry: azuretre
 dockerfile: Dockerfile.tmpl
-required:
-  - docker
 
 credentials:
   - name: azure_tenant_id
-    env: AZURE_TENANT_ID
+    env: ARM_TENANT_ID
```

Comment:
What is reasoning to change from AZURE_* to ARM_*  env var names?

Reply:
The change was made in vanilla - https://github.com/microsoft/AzureTRE/blob/28248ea77d5d94b52c5444b360afccb88546fd62/workspaces/vanilla/porter.yaml#L6 , I believe so that terraform would pick up the variables.

@tompaana can advise further.